api/app/auto_spatial_advisory/sfms.py

```diff
@@ -0,0 +1,83 @@
+""" Code related to managing SFMS output files.
+"""
+import os
+from datetime import date
+from app.schemas.auto_spatial_advisory import SFMSFile, SFMSRunType
+from app.utils.time import get_hour_20, get_vancouver_now
+
+
+def get_date_part(filename: str) -> str:
+    """ Get the date part of the filename.
+    Filename example: hfi20220823.tif
+    """
+    return filename[filename.rfind('.') - 8:filename.rfind('.')]
+
+
+def is_actual(filename: str) -> bool:
+    """ Decide whether the file is an actual or forecast file.
+    08h00 PST on the 22nd hfi20220823.tif -> forecast
+    13h00 PST on the 22nd hfi20220823.tif -> forecast
+    08h00 PST on the 23rd hfi20220823.tif -> forecast
+    13h00 PST on the 23rd hfi20220823.tif -> actual
+    """
+    file_date_string = get_date_part(filename)
+    file_date = date(
+        year=int(file_date_string[:4]),
+        month=int(file_date_string[4:6]),
+        day=int(file_date_string[6:8]))
+    now = get_vancouver_now()
+    now_date = now.date()
+
+    if file_date < now_date:
+        # It's from the past - it's an actual.
+        return True
+    if file_date > now_date:
+        # It's from the future - it's a forecast.
+        return False
+    # It's from today - now it get's weird.
```

Comment:
```suggestion
    # It's from today - now it gets weird.
```

Reply:
what does my 20 points get me???

Reply:
Once you have 100 points, you can trade it for a virtual gold piece!

Reply:
a bitcoin?!?!?

Reply:
Ok fine. I'll get you a bitcoin when it reaches parity with the Zimbabwean dollar.